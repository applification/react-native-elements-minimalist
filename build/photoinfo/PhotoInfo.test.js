'use strict';var _jsxFileName='src/photoinfo/PhotoInfo.test.js';var _react=require('react');var _react2=_interopRequireDefault(_react);var _reactTestRenderer=require('react-test-renderer');var _reactTestRenderer2=_interopRequireDefault(_reactTestRenderer);var _camera=require('react-native-minimalist-icons/camera');var _camera2=_interopRequireDefault(_camera);var _PhotoInfo=require('./PhotoInfo');var _PhotoInfo2=_interopRequireDefault(_PhotoInfo);function _interopRequireDefault(obj){return obj&&obj.__esModule?obj:{default:obj};}var Icon=_react2.default.createElement(_camera2.default,{style:{color:'#000000',height:'1.6em'},__source:{fileName:_jsxFileName,lineNumber:7}});describe('PHOTOINFO',function(){it('Renders photo info component',function(){var component=_reactTestRenderer2.default.create(_react2.default.createElement(_PhotoInfo2.default,{photo:{uri:'https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg'},title:'josh.duplessis',icon:Icon,__source:{fileName:_jsxFileName,lineNumber:12}}));expect(component).toMatchSnapshot();});it('Renders WITHOUT out subtitle',function(){var component=_reactTestRenderer2.default.create(_react2.default.createElement(_PhotoInfo2.default,{photo:{uri:'https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg'},title:'josh.duplessis',icon:Icon,__source:{fileName:_jsxFileName,lineNumber:27}}));expect(component).toMatchSnapshot();});it('Renders WITH a subtitle',function(){var component=_reactTestRenderer2.default.create(_react2.default.createElement(_PhotoInfo2.default,{photo:{uri:'https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg'},title:'bentierney',subtitle:'Guitar Center',icon:Icon,__source:{fileName:_jsxFileName,lineNumber:42}}));expect(component).toMatchSnapshot();});});