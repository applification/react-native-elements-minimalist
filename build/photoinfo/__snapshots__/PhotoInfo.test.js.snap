// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PHOTOINFO Renders WITH a subtitle 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "flex-start",
      "paddingBottom": 5,
      "paddingLeft": 3,
      "paddingTop": 5,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg",
      }
    }
    style={
      Object {
        "borderRadius": 15,
        "height": 30,
        "margin": 7,
        "width": 30,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignContent": "flex-start",
        "alignSelf": "center",
        "flexGrow": 1,
        "paddingLeft": 7,
      }
    }
  >
    bentierney
    

    Guitar Center
  </Text>
  <View
    style={
      Object {
        "alignSelf": "center",
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    <svg
      className="rn-display-4qtqp9 rn-fill-yyyyoo rn-maxWidth-dnmrzs rn-position-bnwqim rn-verticalAlign-m6rgpd rn-userSelect-lrvibr"
      style={
        Object {
          "color": "rgba(0,0,0,1)",
          "height": "1.6em",
        }
      }
      viewBox="0 0 24 24"
    >
      <g>
        <path
          d="M19.75 5.083h-2.188C16.368 3.17 14.278 2 12 2S7.633 3.17 6.438 5.083H4.25C3.01 5.083 2 6.093 2 7.333V19.75C2 20.99 3.01 22 4.25 22h15.5c1.24 0 2.25-1.01 2.25-2.25V7.333c0-1.24-1.01-2.25-2.25-2.25zM12 17.528a4.5 4.5 0 1 1 0-9 4.5 4.5 0 0 1 0 9z"
        />
      </g>
    </svg>
  </View>
</View>
`;

exports[`PHOTOINFO Renders WITHOUT out subtitle 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "flex-start",
      "paddingBottom": 5,
      "paddingLeft": 3,
      "paddingTop": 5,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg",
      }
    }
    style={
      Object {
        "borderRadius": 15,
        "height": 30,
        "margin": 7,
        "width": 30,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignContent": "flex-start",
        "alignSelf": "center",
        "flexGrow": 1,
        "paddingLeft": 7,
      }
    }
  >
    josh.duplessis
  </Text>
  <View
    style={
      Object {
        "alignSelf": "center",
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    <svg
      className="rn-display-4qtqp9 rn-fill-yyyyoo rn-maxWidth-dnmrzs rn-position-bnwqim rn-verticalAlign-m6rgpd rn-userSelect-lrvibr"
      style={
        Object {
          "color": "rgba(0,0,0,1)",
          "height": "1.6em",
        }
      }
      viewBox="0 0 24 24"
    >
      <g>
        <path
          d="M19.75 5.083h-2.188C16.368 3.17 14.278 2 12 2S7.633 3.17 6.438 5.083H4.25C3.01 5.083 2 6.093 2 7.333V19.75C2 20.99 3.01 22 4.25 22h15.5c1.24 0 2.25-1.01 2.25-2.25V7.333c0-1.24-1.01-2.25-2.25-2.25zM12 17.528a4.5 4.5 0 1 1 0-9 4.5 4.5 0 0 1 0 9z"
        />
      </g>
    </svg>
  </View>
</View>
`;

exports[`PHOTOINFO Renders photo info component 1`] = `
<View
  style={
    Object {
      "flexDirection": "row",
      "height": 50,
      "justifyContent": "flex-start",
      "paddingBottom": 5,
      "paddingLeft": 3,
      "paddingTop": 5,
    }
  }
>
  <Image
    source={
      Object {
        "uri": "https://s3.amazonaws.com/uifaces/faces/twitter/kfriedson/128.jpg",
      }
    }
    style={
      Object {
        "borderRadius": 15,
        "height": 30,
        "margin": 7,
        "width": 30,
      }
    }
  />
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "alignContent": "flex-start",
        "alignSelf": "center",
        "flexGrow": 1,
        "paddingLeft": 7,
      }
    }
  >
    josh.duplessis
  </Text>
  <View
    style={
      Object {
        "alignSelf": "center",
        "marginLeft": 10,
        "marginRight": 10,
      }
    }
  >
    <svg
      className="rn-display-4qtqp9 rn-fill-yyyyoo rn-maxWidth-dnmrzs rn-position-bnwqim rn-verticalAlign-m6rgpd rn-userSelect-lrvibr"
      style={
        Object {
          "color": "rgba(0,0,0,1)",
          "height": "1.6em",
        }
      }
      viewBox="0 0 24 24"
    >
      <g>
        <path
          d="M19.75 5.083h-2.188C16.368 3.17 14.278 2 12 2S7.633 3.17 6.438 5.083H4.25C3.01 5.083 2 6.093 2 7.333V19.75C2 20.99 3.01 22 4.25 22h15.5c1.24 0 2.25-1.01 2.25-2.25V7.333c0-1.24-1.01-2.25-2.25-2.25zM12 17.528a4.5 4.5 0 1 1 0-9 4.5 4.5 0 0 1 0 9z"
        />
      </g>
    </svg>
  </View>
</View>
`;
